run_tests.sh

```diff
@@ -178,6 +183,9 @@ elif [[ "$1" == 'test' ]]; then
   files_list=$(find "$TESTS_DIR" | grep --perl-regexp "${regex}" | grep --extended-regexp --invert-match 'samples/.*|/shunit2/')
   set_tests $files_list
   LANGUAGE=en_US.UTF_8 run_tests
+elif [[ "$1" == "--clear-cache" ]]; then
```

Comment:
By the way the conditionals are, the following cases are acceptable
```bash
./run_tests.sh --unit --clear-cache
./run_tests.sh --integration --clear-cache
./run_tests.sh --clear-cache
```
Which one are you aiming to?

IMHO, I think it makes sense to tie the option to `--integration` like in the the second case and maybe have
```bash
./run_tests.sh --integration clear-cache
```
just to keep the pattern of set by the `test`, `list`, and `help` options that come after `--unit`/`--integration`.

Reply:
That is a good idea: to have cached cleared on a scope basis.